sympy/printing/latex.py

```diff
@@ -1710,8 +1715,12 @@ def _print_TensorIndex(self, expr):
         )
 
     def _print_tuple(self, expr):
-        return r"\left( %s\right)" % \
-            r", \  ".join([self._print(i) for i in expr])
+        if self._settings['decimal_separator'] =='comma':
+            return r"\left( %s\right)" % \
+                r"; \  ".join([self._print(i) for i in expr])
+        else:
```

Comment:
Shouldn't else be used for extreme cases which have not been covered instead of just defaulting?

What if someone desires to use ``semicolon`` or ``tilde`` or ``colon`` or something else as a decimal separator. I don't think ignoring all that and giving them default result looks good.

I would suggest to try elif statements for the period as it is the default value anyways and using else for throwing a NotImplementedError or any appropriate error.

Reply:
I've added this now. Just to be clear, do you think I should add a not implemented error for the decimal parts as well(in print float and the empty printer case)? 

Or should I just make the decimal separator and the list separator separate argument inside of print_latex?

Thanks for your suggestions!

Reply:
> I've added this now. Just to be clear, do you think I should add a not implemented error for the decimal parts as well(in print float and the empty printer case)?

Well, you can add it just as a safety measure in case someone gets too adventurous with the separator options but still gets a period. But you'll also have to add an if case for period which effectively does nothing. So, it's your choice.

> Or should I just make the decimal separator and the list separator separate argument inside of print_latex?

I don't think there should be a need for that. The list separator only changes when the decimal separator is a comma.